onnxruntime/core/providers/qnn/builder/qnn_node_group/gelu_fusion.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "core/providers/qnn/builder/qnn_node_group/qnn_node_group.h"
+#include "core/providers/qnn/ort_api.h"
+
+namespace onnxruntime {
+namespace qnn {
+
+class QnnModelWrapper;
+
+/// <summary>
+/// Represents a fusion of the Gelu pattern expanded into ONNX operators.
+/// This fusion handles two patterns:
+/// Pattern 1: root -> Div -> Erf -> Add -> Mul (with Mul from root)
```

Comment:
I don't quite understand this description, in particular, "Mul (with Mul from root)". could it be made clearer, possibly with a more detailed diagram?

Reply:
thanks for adding the diagram, that clears it up for me